plugin/trino-iceberg/pom.xml

```diff
@@ -239,12 +239,32 @@
             <groupId>org.projectnessie.nessie</groupId>
             <artifactId>nessie-client</artifactId>
             <version>${dep.nessie.version}</version>
+            <exclusions>
+                <exclusion>
+                    <groupId>com.fasterxml.jackson.core</groupId>
+                    <artifactId>jackson-annotations</artifactId>
+                </exclusion>
+                <exclusion>
+                    <groupId>com.fasterxml.jackson.core</groupId>
+                    <artifactId>jackson-databind</artifactId>
+                </exclusion>
```

Comment:
Why these exclusions? document

is it safe to exclude these?

Reply:
what is this needed for?

Reply:
Maven enforcer rule was giving a compile error when trino wasn't on the same jackson version. 
Looks like on latest Trino we are on the same version and this is not needed anymore. 

I have removed it now. Thanks for questioning it and making me to try again on the latest version.